### DISCUSSION TITLE
Starter Kit Website

### DISCUSSION THREAD

**Discussion by Bryson69** [SurPathHub/starter-kit#16] [General]
This will be a static website whereby users can easily access resources based on categories. Easy for users, junior or beginner devs to source through. It will contain lots of resources based on different languages. It should be a website that users want to visit more often.

**Comment by vinzvinci** [comment]
Good idea 👍, we will implement this soon. For now, I'll enable the GitHub pages. 

Thank you @Bryson69 :)

**Reply by vinzvinci** [reply]
We need a cool design for this, something unique and accessible for everyone.
